packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -0,0 +1,353 @@
+// @ts-check
```

Comment:
This file is nearly all code from contractFacet.js. I will mark the parts that are different.